caffe2/CMakeLists.txt

```diff
@@ -350,18 +350,25 @@ if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
       "${TORCH_SRC_DIR}/csrc/autograd/generated/ADInplaceOrViewType_0.cpp"
       "${TORCH_SRC_DIR}/csrc/autograd/generated/ADInplaceOrViewType_1.cpp"
     )
+    if(BUILD_LAZY_TS_BACKEND)
```

Comment:
Mostly just my check for understanding - we're going to unconditionally run the LTC codegen and emit the C++ files, right? And then actually building them looks like it will get guarded by this build flag. (I think this seems fine - it's probably not worth the work to make the codegen conditionally emit LTC code if we don't end up building it).

Reply:
Right. The codegen runs always, which we could try to avoid i suppose.

Reply:
(nah seems fine to me)